Convert and clean PDFs using PDFBox

HPrickettMorgan opened the pull request:
@ArthurAllshire This is an alternate implementation to the pull_from_pdf branch which runs faster with less code. Take a look.

Fixes #11 #9 #12, depends on #7.

ArthurAllshire commented on utils/pdf_to_text.sh:
```diff
@@ -0,0 +1,3 @@
+java -jar $PDFBOX_HOME ExtractText $1 "$1.tmp"
```
The arguments to this script should be documented, and it should have the ability to process an entire directory...

HPrickettMorgan replied:
Oof completely didn't mean to push the jar, nice catch. I'll squash it when we merge so that it's not accessible.

HPrickettMorgan replied:
It's pretty simple with just a for loop, I mostly just thought it's too much work for something that can be done externally easy.

ArthurAllshire commented on utils/rats.txt:
```diff
@@ -0,0 +1,138 @@
+RATS on the University of Toronto - St. George Campus
```
What is this file for?

HPrickettMorgan commented:
Added the wrong stuff :(

HPrickettMorgan commented:
@ArthurAllshire I fixed the issues with committing extra files and updated the gitignore so it won't happen again. I think I disagree with processing an entire directory because it would significantly complicate the implementation of the script internally, whereas we can process a directory with something like
`for file in *.pdf; do ./pdf_to_text.sh $file ${file%.pdf}.txt; done`
relatively easily.
